lisa/sut_orchestrator/azure/tools.py

```diff
@@ -237,11 +237,7 @@ def _check_exists(self) -> bool:
 
     def _install(self) -> bool:
         result = self.install_from_iso()
-        if result.exit_code != 0:
-            raise LisaException(
-                f"Unable to install the LIS RPMs! exit_code: {result.exit_code}"
-                f"stderr: {result.stderr}"
```

Comment:
This is not easy to move, because the stderr is not presented in the exception by default. The exception message is more obvious than log, so it's worth to include the stderr. But the default `assert_exit_code` cannot include it, because there may be a lot of stderr/stdout, and doesn't help. So when it needs the stderr, we cannot use the `assert_exit_code`.

Reply:
Should I add a string formatting with `stderr` and `exit_code` when presenting `excepted_exit_code_failure_message`? for example:
```python
excepted_exit_code_failure_message = excepted_exit_code_failure_message % {"stderr": self.stderr, "exit_code": self.exit_code}
```
I saw something like it before, in https://github.com/pallets/click/blob/main/src/click/decorators.py:452

Reply:
The stdout/stderr shouldn't be included by default, because the output may be very long, not readable. Before we find a better way, it's worth to handle separately.